tao_compiler/mlir/disc/transforms/fusion_utils_dot_gpu.cc

```diff
@@ -13,21 +13,72 @@
 #include "tensorflow/compiler/mlir/disc/transforms/fusion_utils.h"
 #include "tensorflow/compiler/mlir/disc/utils/source_emitter.h"
 
+namespace llvm {
+template <>
+struct DenseMapInfo<SmallVector<mlir::Operation*>> {
+  static SmallVector<mlir::Operation*> getEmptyKey() {
+    return SmallVector<mlir::Operation*>{
```

Comment:
just curious: is `SmallVector<mlir::Operation*>`  required? is it OK to just return `{ ... }`